portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -111,11 +111,14 @@ public void displayMaxComments(PreparedQuery results, int maxComment) {
   }
 
   public Comment createCommentFromEntity(Entity commentEntity) {
+
+      System.out.println(commentEntity.getProperty(this.USERNAME_PARAMETER));
```

Comment:
Why this?

Reply:
This was used to troubleshoot the nullPointerExeption that kept occurring when the maxComment display exceeded the comments in the query